StudentData/classes/StudentData.cls

```diff
@@ -0,0 +1,25 @@
+public class StudentData {
+    	public StudentData(){
+		
+		//Insert Student Record
+		Student__c s1 = new Student__c();
+		s1.Name = 'ABC';
+        //Insert Student Record in Teacher Layout
+        Id recordTypeIdOfStudent = Schema.SObjectType.Student__c.getRecordTypeInfosByName().get('Teacher').getRecordTypeId();
+        s1.RecordTypeId = recordTypeIdOfStudent;
+		s1.Roll_Number__c = 2345;
```

Comment:
Indentation is not good. Open the developer console then open this class and click Ctrl+A then click Ctrl+Tab.